### PR TITLE
chore: Pipeline Refactor

### PR DESCRIPTION
**Pull request by williamputraintan**
- Change stage ConstructId name
- Migrate build notification to `alerts-build`
- Change AccountName to AppStage name for consistency

### CODE REVIEW

**Comment by victorskl** [comment]
reviewing...

**Comment by victorskl** [comment]
Ah ok, this one is refactor. At least, #250 #115 merge first. The rest branches have to rebase. Let check with the team.

**Comment by williamputraintan** [comment]
Happy to merge this after those PR are merged!

**Comment by williamputraintan** [comment]
As this require redeployment, I am also happy to wait if anyone has any objection and wanted to postpone this for later.

File: config/config.ts
Comment (brainstorm) [inline config/config.ts:27]: I find single letter parameters unnecessary cryptic? Why not `stage`?

**Review by brainstorm — CHANGES_REQUESTED** [review]

File: config/config.ts
Comment (williamputraintan) [reply config/config.ts]: Yup, updated!

**Review by raylrui — APPROVED** [review]
LGTM!

**Review by victorskl — APPROVED** [review]

**Comment by victorskl** [comment]
@alexiswl This refactor PR can go? And, so that we all follow up rebase onto main after, ok?

**Comment by alexiswl** [comment]
Sure

**Comment by raylrui** [comment]
Hi @williamputraintan the conflict comes from our talk before, of the account id alias.

**Comment by williamputraintan** [comment]
Redeploying ...

**Comment by williamputraintan** [comment]
Yeah, think make sense as well to include here. Will follow the pattern with the [accountId](https://github.com/umccr/orcabus/blob/b88fdb883e64717050fad7b50acb1029aa6bab13/config/constants.ts#L14)

**Pull request merged**
